benefits/working_hours.md

```diff
@@ -0,0 +1,13 @@
+# Working Hours
```

Comment:
Probably worth mentioning that our contractual obligations are 9am to 6pm, how do you resolve conflicts between the contractual with the handbook (which is less so) and verbal (even less so)